### PR TITLE
promoting version 6.0.20-12

### PR DESCRIPTION
**Pull request by yuvallevy2**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -485,11 +478,82 @@ spec:
...
+In order to connect to the web console please use one of the following methods:
+
+Method 1: using port-forward
+1. Get the port of the cluster UI service via the command below, replace the <cluster name> with the name of your Redis Enterprise Cluster:
+```bash
+    kubectl get service/<cluster name>-ui -o yaml
+```
+Note: the default port is 8443.
+![Alt text](./images/web_console_5.png?raw=true)
+2. Run the command below to set port-forward, replace the <cluster name> with the name of your Redis Enterprise Cluster, use the port of the service for the service port and the port you want to use on the local machine as the local port:
```
Comment (kaitlynmichael) [inline README.md:519]:
```suggestion
2. Run the kubectl port-forward service to set port-forward. Replace the <cluster name> with the name of your Redis Enterprise Cluster, replace <service port> with the port of the service, and replace <local port> with the port you want to use on the local machine.
```

File: README.md [hunk 2]
```diff
@@ -485,11 +478,82 @@ spec:
...
+```bash
+    kubectl port-forward service/<cluster name>-ui <local port>:<service port>
+```
+![Alt text](./images/web_console_6.png?raw=true)
+3. In the web browser on the local machine to see the Redis Enterprise web console go to:
```
Comment (kaitlynmichael) [inline README.md:524]:
```suggestion
3. View the web console from the web browser on your local machine: 
```

File: README.md [hunk 3]
```diff
@@ -485,11 +478,82 @@ spec:
...
+https://localhost:<local port>
+Don't forget to replace the <local port> with the one used in the previous command.
+![Alt text](./images/web_console_7.png?raw=true)
+
+Method 2: load balancer
+> Note - configuring a load balancer service for the UI will create an external IP address, widely available (when set on cloud providers which support external load balancers). Use with caution.
```
Comment (kaitlynmichael) [inline README.md:530]:
```suggestion
<note> Configuring a load balancer service for the UI will create an external IP address, widely available (when set on cloud providers which support external load balancers). Use with caution. </note>
```

File: multi-namespace-redb/README.md [hunk 4]
```diff
@@ -58,26 +58,22 @@ roleRef:
...
 * The Redis Enterprise Operator creates a service named after the REDB within the REDB namespace. The service is of type "External Name". This service can only be accessed from within the K8s cluster. Configuring Load Balancer/Node Port services automatically is not supported at this time. The service type configuration within the Redis Enterprise Cluster custom resource will be ignored. The External Name service will point to another ClusterIP service created within the operator namespace, exposing the actual Redis endpoint.
+* When deploying a RedisEnterpriseCluster, the operator will read environment variables from a configmap named operator-environment-config within the operator namespace. Adding new values to this configmap will reset the operator's pod and propagate the new variables to its containers. Note that this configmap can be created manually before deploying the RedisEnterpriseCluster, or the operator will automatically create it once a RedisEnterpriseCluster is deployed.
```
Comment (kaitlynmichael) [inline multi-namespace-redb/README.md:79]:
```suggestion
* When deploying a RedisEnterpriseCluster, the operator will read environment variables from a configmap named `operator-environment-config` within the operator namespace. Adding new values to this configmap will reset the operator's pod and propagate the new variables to its containers. Note that this configmap can be created manually before deploying the RedisEnterpriseCluster, or the operator will automatically create it once a RedisEnterpriseCluster is deployed.
```

**Review by kaitlynmichael — APPROVED** [review]
Just minor comments. Nothing big to correct here.

**Review by kaitlynmichael — APPROVED** [review]

**Review by amiramm — CHANGES_REQUESTED** [review]
LGTM @yuvallevy2 please see minor corrections

**Pull request merged**